src/style.css

```
@@ -0,0 +1,65 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-heavy;
+  src: url(./fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 900;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+.search {
+  display: block;
+  position: relative;
+}
+
+.search__bar {
+  width: 100%;
+  display: block;
+  background: url(./images/Search.svg) 26px 25px no-repeat;
+  font-family: Avenir, sans-serif;
+  font-size: 16px;
+  line-height: 22px;
+  height: 70px;
+  color: #3d4e61;
+  margin: 20px 0;
+  padding: 0 62px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search__small {
```

you will learn about modifiers a bit later (topic BEM), but if the classname adds only slight changes into the behavior of the existing selector you should use --
```suggestion
.search--small {
```